lib/vsc/utils/mail.py

```diff
@@ -68,8 +69,25 @@ def __init__(self, mail_host=None, mail_to=None, mail_from=None, mail_subject=No
 class VscMail(object):
     """Class providing functionality to send out mail."""
 
-    def __init__(self, mail_host=None):
-        self.mail_host = mail_host
+    def __init__(self, mail_host=None, auth=False, mail_config=None):
+
+        opts = {
+            "from": ("Sender", None, "store", None),
+            "smtp": ("Host:Port", None, "store", "exploud.ugent.be:587"),
+            "smtp-auth-password": ("Authentication password", None, "store", None),
+            "smtp-auth-user": ("Authentication user", None, "store", None),
+            "smtp-use-starttls": ("Should we start a TLS session", bool, "store_true", False),
+        }
+
+        self.auth = auth
+
+        if self.auth:
+            self.go = simple_option(opts, config_files=mail_config)
+            (host, port) = self.go.options.smtp.split(":")
+            self.mail_host = host
```

Comment:
it's better to set these always, and default for host can be `''` and port `0`. it also simplifies the code below